sammba/registration/coregistrator.py

```diff
@@ -111,8 +111,8 @@ def fit_modality(self, in_file, modality, slice_timing=True,
                 if t_r is None:
                     raise ValueError("'t_r' is needed for slice timing "
                                      "correction")
-                func_file = _slice_time(in_file, t_r, self.output_dir,
-                                        caching=self.caching)
+                func_file = _slice_time(in_file, t_r, time_pattern='altplus',
+                                        self.output_dir, caching=self.caching)
```

Comment:
you can not put a function optional parameter before a mandatory parameter. This is the reason for travis error
>SyntaxError: non-keyword arg after keyword arg (template_registrator.py, line 235)